### PR TITLE
gir1.2-notify-0.7 dependency

### PR DESCRIPTION
**Pull request by lene**
This dependency was introduced somewhere but not added to debian/control. Fixing this.

### CODE REVIEW

File: test/integration/nicotine.robot [hunk 1]
```diff
@@ -11,5 +11,5 @@ Library           nicotine_library.py
 
 Run nicotine as shell command
     Running nicotine starts a process    nicotine    3
-    Result should be    0
+    Result should be    ${True}
```
Comment (kiplingw) [inline test/integration/nicotine.robot:14]: Thanks @lene. Ignore my incompetence. I'm not very familiar with the Robot Framework.

**Comment by kiplingw** [comment]
Thanks @lene. When I was first introduced to Debian's `autopkgtest` it all seemed like overkill for testing. Then it took enough situations like this to realize where it shines because your development environment always contains all kinds of side effects and alterations that's impossible to control for when trying to replicate an end user's environment if you don't have a hermetically sealed container.

File: debian/control [hunk 2]
```diff
@@ -26,6 +26,7 @@ Depends:
     miniupnpc (>= 1.9),
     gobject-introspection,
     gir1.2-gtk-3.0,
+    gir1.2-notify-0.7,
```
Comment (kiplingw) [inline debian/control:29]: Should we add this to README.md as well? I'm not sure if this is a Debian and derivative specific nomenclature or if this package is more generally known by this name in the wild.

Comment (lene) [reply debian/control:29]: Probably a good idea! I know nothing about this package, but it looks awfully specific.

**Comment by lene** [comment]
> Thanks @lene. When I was first introduced to Debian's `autopkgtest` it all seemed like overkill for testing. Then it took enough situations like this to realize where it shines because your development environment always contains all kinds of side effects and alterations that's impossible to control for when trying to replicate an end user's environment if you don't have a hermetically sealed container.

Yes, I must admit I'm a convert! I'll still moan about what a pain it is to set up though.

**Comment by kiplingw** [comment]
It is definitely a pain to setup the first time. But after that it's pretty easy to set them up for different releases and different architectures like `arm64`, `ppc64el`, etc.

File: debian/control (hunk 2 above)
Comment (mathiascode) [reply debian/control:29]: At least on Manjaro/Arch, everything seems to be bundled in one "libnotify" package.

Comment (kiplingw) [reply debian/control:29]: I'll add it anyways for safe measure.

**Pull request merged**